pkg/daemon/ceph/client/radosnamespace.go

```diff
@@ -104,3 +104,32 @@ func DeleteRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, p
 	logger.Infof("successfully deleted rados namespace %s/%s in k8s namespace %q", poolName, namespaceName, clusterInfo.Namespace)
 	return nil
 }
+
+// EnableRBDRadosNamespaceMirroring enables rbd mirroring on a rados namespace.
+func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespaceName, remoteNamespace, mode string) error {
+	logger.Infof("enable mirroring in rados namespace %s/%s in k8s namespace %q", poolName, namespaceName, clusterInfo.Namespace)
+	args := []string{"mirror", "pool", "enable", "--pool", poolName, "--namespace", namespaceName, mode, "--remote-namespace", remoteNamespace}
+	cmd := NewRBDCommand(context, clusterInfo, args)
+	cmd.JsonOutput = false
+	output, err := cmd.Run()
+	if err != nil {
+		return errors.Wrapf(err, "failed to enable mirroring in rados namespace %s/%s. %s", poolName, namespaceName, output)
+	}
+
+	logger.Infof("successfully enabled mirroring in rados namespace %s/%s in k8s namespace %q", poolName, namespaceName, clusterInfo.Namespace)
```

Comment:
why do we need to log clusterNamespace instead of clusterName or clusterName+Namespace?

Reply:
There is only one cephcluster CR allowed per namespace, so the namespace is typically only logged in rook